sunpy/net/tests/test_fido.py

```diff
@@ -124,6 +124,16 @@ def test_save_path_pathlib(tmpdir):
         assert f"eve{os.path.sep}0" in f
 
 
+@pytest.mark.remote_data
+def test_save_path_cwd(tmpdir):
+    qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
+
+    # Test when path is ./ for current working directory
+    os.chdir(tmpdir)  # move into temp directory
```

Comment:
Do we want to exit the directory after the test is done?

Reply:
I was gonna hope/pray that it doesn't matter since the test is done and it all resets ok.

Reply:
Is there not a way to do directory changes with pytest to ensure that nothing blows up?

Reply:
:shrug:

Reply:
its all a mystery ...

Reply:
The tests don't seem to have blown up so shall we just merge this?

Reply:
Yeah